ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/helpers/AndroidApplicationContext.kt

```diff
@@ -13,20 +14,31 @@ internal interface ApplicationContext {
     fun getLocale(): Locale
 }
 
-internal class AndroidApplicationContext(private val applicationContext: Context) : ApplicationContext {
+internal class AndroidApplicationContext(
```

Comment:
This is not an application context anymore... How about renaming to `ResourceProvider` or something like that?  We do have the application name as well, but I think that's ok.